src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -74,6 +71,16 @@ const Body = BaseForm.extend(Object.assign(
     remove () {
       BaseForm.prototype.remove.apply(this, arguments);
       this.stopPolling();
+    },
+
+    triggerAfterError (model, error) {
+      BaseForm.prototype.triggerAfterError.apply(this, arguments);
+      this.stopPolling();
+
+      // Polling needs to be resumed if it's a form error and session is still valid
+      if(error.responseJSON.errorMessageKey !== 'idx.session.expired') {
+        this.startPolling();
```

Comment:
Does session expire when we hit ratelimit ?

Reply:
It hits the rate limit after session expires.. before that it works fine as expected.

Reply:
nvm